core/kubernetes/dev/dev-urls.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+res="$(kubectl get service openhim-mongo-dev-service)"
+
+if [ "$?" == "0" ]; then
+  url="$(minikube service openhim-mongo-dev-service --url)"
+
+  echo -e "The mongo url:\n $url\n"
+else
+  echo "Mongo dev not started (in dev mode)"
+fi
+
+res1="$(kubectl get service hapi-fhir-mysql-dev-service)"
```

Comment:
We might need to replace the protocol (`http://`) from this result as the database is accessed via tcp and not http